### PR TITLE
Refactored !help descriptions for all scripts and removed a few unused/redundant scripts

### PR DESCRIPTION
**Pull request by mitchmcdee**
This PR mainly tidies up !help descriptions for all scripts to provide a more concise and standardised experience.

Complete summary of changes:
- Rewrote (nearly) every script to reflect a standardised !help description
- Removed unnecessary 3rd party hubot scripts (translate, maps, google, better-voting, images, youtube) as they are (I think) unused and significantly cluttered up !help
- Removed conversation dependency and associated script as it seemed pointless
- Removed all coffee scripts and rewrote JavaScript equivalents for go-down.coffee and help.coffee
- Updated admin usernames to reflect the new committee
- Removed radar.js as the functionality is found in weather anyway
- Removed reaction.js as it was old, half baked and unused
- Reduced uqfinal commands to just !uqfinal as the others didn't seem necessary

Happy to make and edits/updates/clarifications!

### CODE REVIEW

**Comment by mitchmcdee** [comment]
image

What the new !help looks like

**Comment by nickl93** [comment]
Changes all look good to me, though I would suggest including !radar, at least as an alias of “!weather radar” as !radar is the one used/known by most people in UQCS.

File: external-scripts.json [hunk 1]
```diff
@@ -1,10 +1,4 @@
 [
-  "hubot-google-translate",
-  "hubot-maps",
-  "hubot-google",
   "hubot-postgres-brain",
-  "hubot-better-voting",
-  "hubot-google-images",
-  "hubot-youtube",
```
Comment (TRManderson) [inline external-scripts.json]: Youtube is used fairly frequently in #music, and occasionally in other channels

File: package.json [hunk 2]
```diff
@@ -7,20 +7,13 @@
...
-    "hubot-google-images": "^0.2.3",
-    "hubot-google-translate": "^0.2.0",
     "hubot-help": "^0.1.2",
-    "hubot-maps": "0.0.2",
     "hubot-postgres-brain": "^0.1.0",
     "hubot-rules": "^0.1.1",
     "hubot-scripts": "^2.16.2",
     "hubot-slack": "^4.0.0",
     "hubot-wolfram": "1.0.9",
-    "hubot-youtube": "^1.0.1",
```
Comment (TRManderson) [inline package.json]: As above

File: scripts/events.js [hunk 3]
```diff
@@ -2,7 +2,7 @@
 //   Parses the UQCS calendar (https://calendar.google.com/calendar/ical/q3n3pce86072n9knt3pt65fhio%40group.calendar.google.com/public/basic.ics)
 //
 // Commands:
-//   !`events` _<description of time>_ - List UQCS Events
+//   `!events [full|all|NUM EVENTS] [<NUM WEEKS> weeks]` - List UQCS Events
```
Comment (TRManderson) [inline scripts/events.js]: I'd prefer `[full|all|NUM EVENTS|<NUM WEEKS> weeks]` given you can't do `full 2 weeks`

File: scripts/example.coffee [hunk 4]
```diff
@@ -1,108 +0,0 @@
...
-#
-# Notes:
-#   They are commented out by default, because most of them are pretty silly and
-#   wouldn't be useful and amusing enough for day to day huboting.
-#   Uncomment the ones you want to try and experiment with.
-#
-#   These are from the scripting documentation: https://github.com/github/hubot/blob/master/docs/scripting.md
-
-module.exports = (robot) ->
-  robot.respond /repo/i, (res) ->
```
Comment (TRManderson) [inline scripts/example.coffee]: Given that removing this also removes `!repo`, and there's no harm in having some trivial default examples, I'd rather keep this one or at the very least, replace it with a script just for `!repo`?

File: scripts/exec.js [hunk 5]
```diff
@@ -2,8 +2,8 @@
 //   Shell for UQCSBot (allows advanced use of commands)
 //
 // Commands:
-//   !`shell` _<command>_ - Executes the shell command
-//
+//   `!exec <COMMAND>` - Executes the shell command
```
Comment (TRManderson) [inline scripts/exec.js]: We really need to get @lewisjb to come up with a better description for this

File: scripts/shutdown.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+//   Shutdown uqcsbot
+//
+// Commands:
+//   `!uqcsbot shutdown` - Shuts down uqcsbot, waiting to be restarted via CESI
+
+var admin = ['trm', 'mcdee', 'csa', 'guthers', 'artemis', 'rob', 'mb'];
+
+module.exports = function (robot) {
+    robot.respond(/!?uqcsbot shutdown/i, function (res) {
+        comandee = res.message.user.name
```
Comment (TRManderson) [inline scripts/shutdown.js]: Can't we use `res.message.user.is_admin`?

File: scripts/radar.js [hunk 7]
```diff
@@ -1,11 +0,0 @@
-// Description
```
Comment (TRManderson) [inline scripts/radar.js]: While the functionality for this is technically duplicated in weather.js, it's hella broken, so can we keep this one?

File: scripts/ping.js [hunk 8]
```diff
@@ -4,8 +4,9 @@
 //   This command works for anyone in channels with less that 50 people
 //   
 // Commands:
-//   !`ping` (channel|everyone|here)
-var allowed = ['trm', 'dmarj97', 'csa', 'mqt', 'ainz', 'gricey', 'cat'];
+//   `!ping (channel|everyone|here)` - Pings channel/everyone/here only if the user is an admin OR the channel contains less than 50 people
+
+var allowed = ['trm', 'mitch', 'csa', 'guthers', 'artemis', 'rob', 'mb'];
```
Comment (TRManderson) [inline scripts/ping.js]: Again we can probably use `res.message.user.is_admin`

File: scripts/exec.js (hunk 5 above)
Comment (mitchmcdee) [reply scripts/exec.js]: yeah I really didn't know what to do with it so just fixed the wrong word and decided it was for another day bahahahaha

File: external-scripts.json (hunk 1 above)
Comment (mitchmcdee) [reply external-scripts.json]: Done.

File: package.json (hunk 2 above)
Comment (mitchmcdee) [reply package.json]: Done.

File: scripts/events.js (hunk 3 above)
Comment (mitchmcdee) [reply scripts/events.js]: Done

File: scripts/example.coffee (hunk 4 above)
Comment (mitchmcdee) [reply scripts/example.coffee]: Ah totally missed the fact this has !repo, will write one now :)

As for example scripts, I agree with the conversation on slack in that the repo already contains many great and simple examples for writing scripts, so I don't think a dedicated example file is necessary, but maybe that's something we can look to building in future :)

Comment (mitchmcdee) [reply scripts/example.coffee]: Done

File: scripts/ping.js (hunk 8 above)
Comment (mitchmcdee) [reply scripts/ping.js]: Done

File: scripts/shutdown.js (hunk 6 above)
Comment (mitchmcdee) [reply scripts/shutdown.js]: Done

File: scripts/radar.js (hunk 7 above)
Comment (mitchmcdee) [reply scripts/radar.js]: Done

**Comment by mitchmcdee** [comment]
All requested changes made :) PTAL @TRManderson

**Pull request merged**